src/index.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const program = require('caporal');
+const chalk = require('chalk');
+
+const projectConfig = require('../package.json');
+const dockerClient = require('./docker-client');
+const logger = require('./logger');
+const serviceChecks = require('./service-checks');
+
+process.on('uncaughtException', (err) => {
+  logger.error(err.message);
+  process.exit(2);
+});
+
+process.on('unhandledRejection', (err) => {
+  logger.error(err.message);
+  process.exit(2);
+});
+
+const check = async ({ serviceName }, options) => {
+  if(serviceName) {
+    const service = await dockerClient.getService(serviceName).inspect();
+    process.exit(await checkService(service, options));
+  } else {
+    process.exit(await checkAllServices(options));
```

Comment:
It probably wouldn't be an issue with the single service check, but we may need to cap the error count here to 255 since bash still uses 8-bit return codes.

Reply:
Good catch, that has been fixed